code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -103,6 +103,77 @@
 	icon_state = "bottle"
 	preloaded_reagents = list("frostoil" = 60)
 
+// === Used by trade stations
+/obj/item/reagent_containers/glass/bottle/carpotoxin
+	name = "carpotoxin bottle"
+	desc = "A small bottle. Contains carpotoxin."
+	icon_state = "bottle"
+	preloaded_reagents = list("carpotoxin" = 60)
+	spawn_blacklisted = TRUE	// These were made for the trade program. Blacklisting until otherwise to prevent unintended consequences.
+
+/obj/item/reagent_containers/glass/bottle/pararein
+	name = "pararein bottle"
+	desc = "A small bottle. Contains pararein."
+	icon_state = "bottle"
+	preloaded_reagents = list("pararein" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/aranecolmin
+	name = "aranecolmin bottle"
+	desc = "A small bottle. Contains aranecolmin."
+	icon_state = "bottle"
+	preloaded_reagents = list("aranecolmin" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/blattedin
+	name = "blattedin bottle"
+	desc = "A small bottle. Contains blattedin."
+	icon_state = "bottle"
+	preloaded_reagents = list("blattedin" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/diplopterum
+	name = "diplopterum bottle"
+	desc = "A small bottle. Contains diplopterum."
+	icon_state = "bottle"
+	preloaded_reagents = list("diplopterum" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/seligitillin
+	name = "seligitillin bottle"
+	desc = "A small bottle. Contains seligitillin."
+	icon_state = "bottle"
+	preloaded_reagents = list("seligitillin" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/starkellin
+	name = "starkellin bottle"
+	desc = "A small bottle. Contains starkellin."
+	icon_state = "bottle"
+	preloaded_reagents = list("starkellin" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/gewaltine
+	name = "gewaltine bottle"
+	desc = "A small bottle. Contains gewaltine."
+	icon_state = "bottle"
+	preloaded_reagents = list("gewaltine" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/fuhrerole
+	name = "fuhrerole bottle"
+	desc = "A small bottle. Contains fuhrerole."
+	icon_state = "bottle"
+	preloaded_reagents = list("fuhrerole" = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/reagent_containers/glass/bottle/kaiseraurum
+	name = "kaiseraurum bottle"
+	desc = "A small bottle. Contains kaiseraurum."
+	icon_state = "bottle"
+	preloaded_reagents = list("kaiseraurum" = 60)
+	spawn_blacklisted = TRUE
+// ===
```

Comment:
Ok since there are not a ton of bottles. Otherwise we could make a glass/bottle/trade subtype that automatically adapts its name and description based on the preloaded reagent. That way you would just have something like

/obj/item/reagent_containers/glass/bottle/trade/pararein
	preloaded_reagents = list("pararein" = 60)

instead of repeating 5 lines that are basically the same.

Reply:
Since I'm adding more bottles in the next pass, maybe it would be cleaner to have the trade station take the reagent datum and create a new bottle object with it. Then, I wouldn't have to define a new object every time I want to add a new reagent to the list.